app/models/message.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: messages
+#
+#  id         :bigint           not null, primary key
+#  text       :text             not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint           not null
+#
+# Indexes
+#
+#  index_messages_on_user_id  (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+class Message < ApplicationRecord
+  validates :user_id, presence: true
+  validates :text, presence: true
+  belongs_to :user
```

Comment:
Please use correct order

Reply:
Side note: there isn't a solid correct order until you establish a team convention. The most important part is staying consistent with the rest of the existing models and [using the common approaches](https://stackoverflow.com/questions/25768978/common-method-order-for-rails-models).